<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_tray_uuid">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan Tray ID</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>391</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Create Bar </string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>69</y>
       <width>371</width>
       <height>85</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Bar Name</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_barName">
        <property name="text">
         <string>BARXXXXXX</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>MF Bar UUID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_mf_bar_uuid"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>ALS Bar UUID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_als_bar_uuid"/>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="pushButton_create_bar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Create New Bar Object</string>
     </property>
    </widget>
   </widget>
   <widget class="QComboBox" name="comboBox_select_barpos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>50</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bar Pos.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>88</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dist. (mm)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_enter_mm">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thin Film</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_select_thinfilm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_to_bar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>260</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to
Bar Layout</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>391</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Add Thin Films</string>
    </property>
    <widget class="QLabel" name="label_tray_name">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>36</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>TRAYXXXXX</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_upload_bar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>500</y>
      <width>231</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Upload Bar Layout to Database</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_print_barcode">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>340</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Print Barcode</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>341</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Bar Layout</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>468</x>
      <y>30</y>
      <width>311</width>
      <height>452</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>mm </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Thin Film Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_1"/>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_1"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_2"/>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_3"/>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_3"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_4"/>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_4"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_5"/>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_5"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_6"/>
     </item>
     <item row="6" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_6"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_7"/>
     </item>
     <item row="7" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_7"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_8"/>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_8"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_9"/>
     </item>
     <item row="9" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_9"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_10"/>
     </item>
     <item row="10" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_10"/>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_11"/>
     </item>
     <item row="11" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_11"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_12"/>
     </item>
     <item row="12" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_12"/>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_13"/>
     </item>
     <item row="13" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_13"/>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBox_mm_14"/>
     </item>
     <item row="14" column="2">
      <widget class="QLineEdit" name="lineEdit_tf_14"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGIWAXS_BAR_CREATOR">
    <property name="title">
     <string>GIWAXS BAR CREATOR</string>
    </property>
   </widget>
   <addaction name="menuGIWAXS_BAR_CREATOR"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
